Packs/CiscoEmailSecurity/Integrations/CiscoEmailSecurity/CiscoEmailSecurity.yml

```diff
@@ -0,0 +1,1220 @@
+      description: The sender base score host of the message.
+      type: String
+    - contextPath: CiscoEmailSecurity.Message.showAMP
+      description: Is the AMP shown.
+      type: Boolean
+    - contextPath: CiscoEmailSecurity.Message.showDLP
+      description: Is the DLP shown.
+      type: Boolean
+    - contextPath: CiscoEmailSecurity.Message.showSummaryTimeBox
+      description: Is the summary time box shown.
+      type: Boolean
+    - contextPath: CiscoEmailSecurity.Message.showURL
+      description: Is the URL shown.
+      type: Boolean
+    - contextPath: CiscoEmailSecurity.Message.smtpAuthId
+      description: The SMTP auth ID of the message.
+      type: String
+    - contextPath: CiscoEmailSecurity.Message.subject
+      description: The message date subject.
+      type: String
+    - contextPath: CiscoEmailSecurity.Message.summary.description
+      description: The message summary description.
+      type: String
+    - contextPath: CiscoEmailSecurity.Message.summary.lastEvent
+      description: The message summary last event.
+      type: Number
+    - contextPath: CiscoEmailSecurity.Message.summary.timestamp
+      description: The message summary timestamp.
+      type: String
+    - contextPath: CiscoEmailSecurity.Message.timestamp
+      description: The message date time.
+      type: String
+  - arguments:
+    - default: false
+      description: Date from which the search begins. Template for date - YYYY-MM-DD
+        hh:mm:00
+      isArray: false
+      name: start_date
+      required: true
+      secret: false
+    - default: false
+      description: Date the search ends. Template for date - YYYY-MM-DD hh:mm:00
+      isArray: false
+      name: end_date
+      required: true
+      secret: false
+    - default: false
+      description: |-
+        Specify the number of records to retrieve.
+        If you use a limit you should also use a offset.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Specify an offset value to retrieve a subset of records starting with the offset value.
+        If you use a offset you should also use a limit.
+      isArray: false
+      name: offset
+      required: false
+      secret: false
+    - default: false
+      description: Address to filter with.
+      isArray: false
+      name: order_by_from_address
+      required: false
+      secret: false
+    - default: false
+      description: Address to filter with.
+      isArray: false
+      name: order_by_to_address
+      required: false
+      secret: false
+    - default: false
+      description: Subject to filter with.
+      isArray: false
+      name: order_by_subject
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Address order filtering.
+      isArray: false
+      name: order_dir_from_address
```

Comment:
unclear name

Reply:
@ChanochShayner Did you see this?

Reply:
Done, From another test I did, according to the swagger, I changed the above arguments.